MaxText/input_pipeline/_input_pipeline_utils.py

```diff
@@ -68,6 +69,37 @@ def add_segmentation_and_position(x, data_columns, padding_token=0):
 ########## Functions used by HF pipeline
 
 
+def reformat_prompt(example, column, model_name):
+  """reformat prompt for multimodal SFT"""
+  example[column] = multimodal_utils.reformat_prompt(example[column], model_name)
+  return example
+
+
+def reformat_response(example, column, model_name):
+  """reformat response for multimodal SFT"""
+  example[column] = multimodal_utils.reformat_response(example[column], model_name)
+  return example
+
+
+def pre_process_image_sft(example, image_column, model_name):
+  """pre-process image for multimodal SFT"""
+  image = multimodal_utils.convert_to_RGB(example[image_column])
+  # TODO(aireenmei, hengtaoguo): add support for different image sizes
```

Comment:
Will this be model specific? i.e. a config to control if we should resize the images?